src/components/MonthlyGood/MonthlyGoodSelectorMobile.vue

```diff
@@ -29,58 +49,55 @@
 						'click-mgpromo-cause',
 						option.marketingName
 					]"
+					class="tw-w-full tw-flex tw-items-center tw-mb-1.5"
 					@click="selectCause(option)"
 				>
 					<img
-						class="monthly-selector-mobile__causes-icon"
+						class="tw-h-5 tw-w-5 tw-rounded-sm tw-overflow-hidden tw-mr-1"
 						:src="getImage(`./mg-${option.value}.svg`)"
 					>
 					{{ option.marketingName }}
 				</button>
 			</div>
-			<div class="monthly-selector-mobile__amounts" v-if="lightboxStep == 'amount'">
-				<button
-					v-for="(option, index) in mgAmountOptions"
-					:key="index"
-					v-kv-track-event="[
-						'homepage',
-						'click-mgpromo-amount',
-						option.value
-					]"
-					@click="selectAmount(option.value)"
+			<div class="tw-mt-2" v-if="lightboxStep === 'amount'">
+				<h4 class="tw-py-1">
```

Comment:
This should probably be `<label class="tw-text-h4 tw-py-1" for="mgAmountDropdown">` so the select has a label